.github/workflows/zero_shot_classification.yml

```diff
@@ -0,0 +1,39 @@
+name: zero_shot_classification
+
+on:
+  pull_request: 
+    branches:
+      - master # this can be main
+    paths:
+      - "zero_shot_classification/**"
+
+jobs:
+  zero_shot_classification:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./zero_shot_classification
```

Comment:
Rename dir to `zero-shot-classification` and change it everywhere. All pipeline names will have this format